jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard-view.js

```diff
@@ -69,6 +69,14 @@ define([
     }
 
     PolymerSupport.init();
+    
+    // register delegate handlers that persist across dashboard mode changes  
+    $('#notebook_panel').on('change', '.show-code-label input', function() {
+        $('#notebook-container').toggleClass('hide-code', !this.checked);
+    });
+    $('#notebook_panel').on('click', '.more-info-link', function() {
+        $(this).parents('.help-area').toggleClass('show-more', 200);
+    });
```

Comment:
I was thinking if there is a jQuery function to remove all namespaced events on all child elements. If that doesn't exist, then the elements do need to be known.